README.md

```diff
@@ -681,7 +695,7 @@ task to your `Taskfile.yml` for the first time).
     cmds:
       - task: pantheon:fetch-db
       - task: drupal:import-db
-  sync:tugboat:
+  tugboat:sync:
```

Comment:
Let's address the <service:action> changes for a future issue. Remove this for now.